docs/usage.md

```diff
@@ -92,14 +93,15 @@ All these values, if `undefined`, fallback to the scopes described in [option re
 
 | Name | Description
 | ---- | ----
+| `sumKeys` | Define multiple keys to add additional sums, on top of the `key` one, for scriptable options use.
```

Comment:
Is there a reason to have this first? If not, order alphabetically

Reply:
No reason... I forgot to move it when I have changed the option name :( Sorry